apps/web/src/app/components/Wallet/CurrencyLogoWithNetwork.tsx

```diff
@@ -0,0 +1,51 @@
+import { XChainId } from 'app/pages/trade/bridge/types';
+import React from 'react';
+import styled from 'styled-components';
+import CurrencyLogo from '../CurrencyLogo';
+import { Currency } from '@balancednetwork/sdk-core';
+import { ChainLogo } from 'app/pages/trade/bridge/_components/ChainLogo';
```

Comment:
Could you move the `CurrencyLogoWithNetwork` component in `app/components` folder?
move the `app/pages/trade/bridge/_components/ChainLogo` component in `app/components` folder.
The purpose is to move the shared components in `app/components` folder.